src/reducers/app.js

```diff
@@ -0,0 +1,18 @@
+import { MENU_SATE_CHANGE, REDIRECT_FROM_PAGE } from '../action/HeaderActions'
+
+const initialState = () => {
+    return {
+
+    }

```

Comment:
You should add comment why you added empty initial state. For instance '//TODO it will be change in the future.